docs/content/guides/dagster/recommended-project-structure.mdx

```diff
@@ -0,0 +1,204 @@
+│   │   │   ├── __init__.py
+│   │   │   └── activity_forecast.py
+│   │   ├── core
+│   │   │   ├── __init__.py
+│   │   │   ├── id_range_for_time.py
+│   │   │   └── items.py
+│   │   └── recommender
+│   │       ├── __init__.py
+│   │       ├── comment_stories.py
+│   │       ├── recommender_model.py
+│   │       ├── user_story_matrix.py
+│   │       └── user_top_recommended_stories.py
+        ....
+```
+
+---
+
+## For schedules and sensors
+
+In this example, we put sensors and schedules together in the [`sensors`](https://github.com/dagster-io/dagster/blob/master/examples/project_fully_featured/project_fully_featured/sensors) folder. When we build sensors, they are considered policies for when to trigger a particular job. Keeping all the policies together helps us understand what what's available when creating jobs.
+
+**Note:** Certain sensors, like [run status sensors](/concepts/partitions-schedules-sensors/sensors#run-status-sensors), can listen to multiple jobs and do not trigger a job. We recommend keeping these sensors in the definition as they are often for alerting and monitoring at the code location level.
+
+---
+
+## For resources
+
+Make resources reusable and share them across jobs or asset groups.
+
+In this example, we grouped resources (e.g., database connections, Spark sessions, API clients, and I/O managers) in the [`resources`](https://github.com/dagster-io/dagster/blob/master/examples/project_fully_featured/project_fully_featured/resources/) folder, where they are bound to configuration sets that vary based on the environment.
+
+In complex projects, we find it helpful to make resources reusable and configured with pre-defined values via <PyObject object="configured" />. This approach allows your teammates to use a pre-defined resource set or make changes to shared resources, thus enabling more efficient project development.
+
+This pattern also helps you easily execute jobs in different environments without code changes. In this example, we dynamically defined a code location based on the deployment in [`__init__.py`](https://github.com/dagster-io/dagster/blob/master/examples/project_fully_featured/project_fully_featured/\__init\_\_.py) and can keep all code the same across testing, local development, staging, and production. Read more about our recommendations in the [Transitioning data pipelines from Development to Production](/guides/dagster/transitioning-data-pipelines-from-development-to-production) guide.
+
+---
+
+## For jobs
+
+When using asset-based data pipelines, we recommend having a `jobs.py` file that imports the assets, partitions, sensors, etc. to build each job.
+
+### When using ops and graphs
+
+This project does not include ops or graphs, if it did, this would be the recommendation on how to structure it.
+
+In these situations, we recommend having a jobs folder rather than a jobs.py file. Depending on the types of jobs you have, you can create a seperate file for each types of jobs.
```

Comment:
```suggestion
In these situations, we recommend having a `jobs` folder rather than a `jobs.py` file. Depending on the types of jobs you have, you can create a separate file for each types of jobs.
```